scripts/fiber/tests-failing.txt

```diff
@@ -271,6 +250,7 @@ src/renderers/dom/shared/wrappers/__tests__/ReactDOMInput-test.js
 * should control radio buttons
 * should control radio buttons if the tree updates during render
 * should warn with value and no onChange handler and readOnly specified
+* should have a this value of undefined if bind is not used
```

Comment:
I think the error happens now because the event fires, so the expect does, but not yet sure why it isn't dispatched correctly.